examples/byoi/ubuntu-fips/README.md

```diff
@@ -19,3 +22,15 @@ install:
 Notes:
 - The dracut patch is needed as Ubuntu has an older version of systemd
 - Most of the Dockerfile configuration are: packages being installed by Ubuntu, and the framework files coming from Kairos containing FIPS-enabled packages
+- The LiveCD is not running in fips mode
```

Comment:
Maybe this one deserves to go in an alert

Reply:
mmm but it's the README, not the website

Reply:
my bad hehe